backend/mempool-config.test.json

```diff
@@ -0,0 +1,163 @@
+{
+  "MEMPOOL": {
+    "OFFICIAL": false,
+    "NETWORK": "mainnet",
+    "BACKEND": "none",
+    "ENABLED": true,
+    "HTTP_PORT": 8999,
```

Comment:
I'd suggest setting a different port here than the default, otherwise I have to stop my local mempool backend when running these tests to avoid
```
uncaughtException: Error: listen EADDRINUSE: address already in use :::8999
```